gulpfile.js

```diff
@@ -1,46 +1,68 @@
 /* global require */
+
 var gulp = require('gulp-npm-run')(require('gulp'), {
       exclude: ['test'],
       require: ['doc']
     }),
-    del = require('del'),
+    del = require('del')
+    bump = require('gulp-bump'),
     qunit = require('gulp-qunit'),
     jshint = require('gulp-jshint'),
     sass = require('gulp-sass'),
     concat = require('gulp-concat'),
     uglify = require('gulp-uglify'),
+    replace = require('gulp-replace'),
     rename = require('gulp-rename'),
-    replace = require('gulp-replace');
+    minimist = require('minimist'),
+    deploy = require('gulp-gh-pages-cname');
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>'deploy' is not defined.